docs/src/component-statuses.js

```diff
@@ -0,0 +1,67 @@
+import {Link} from '@primer/components'
+import StatusLabel from '@primer/gatsby-theme-doctocat/src/components/status-label'
+import Table from '@primer/gatsby-theme-doctocat/src/components/table'
+import {graphql, Link as GatsbyLink, useStaticQuery} from 'gatsby'
+import React from 'react'
+
+export function ComponentStatuses() {
+  const data = useStaticQuery(graphql`
+    query ComponentStatuses {
+      allMdx(
+        filter: {frontmatter: {status: {ne: null}}}
+        sort: {fields: frontmatter___title}
+      ) {
+        nodes {
+          slug
+          excerpt(pruneLength: 300)
+          frontmatter {
+            status
+            title
+          }
+        }
+      }
+    }
+  `)
+
+  const components = data.allMdx.nodes.map((node) => {
+    // get first sentence of the excerpt
+    // search for a period space and capital letter to avoid splitting at e.g.
+    const match = node.excerpt
+      .replaceAll('\n', ' ')
+      .match(/(^.+?\.)(\s+[A-Z])/, '.')
```

Comment:
This is a bit sketchy. Would we want to have descriptions defined separately? I thought about updating the `docs.rake` script to add a frontmatter description to each of the md files but that would have taken me a lot longer than doing it this way 😅 and have similar sanitisation issues as this... thoughts welcome